libkbfs/folder_branch_ops.go

```diff
@@ -2135,7 +2149,7 @@ func (fbo *folderBranchOps) createEntryLocked(
 }
 
 func (fbo *folderBranchOps) doMDWriteWithRetry(ctx context.Context,
-	lState *lockState, fn func(lState *lockState) error) error {
+	lState *lockState, waitForCR bool, fn func(lState *lockState) error) error {
```

Comment:
I was just thinking you could do the wait only if you got an `ExclOnUnmergedError`, without needing to pass any additional flags.  Any reason that wouldn't work?